test/translator/read_capacity_10_test.cc

```diff
@@ -0,0 +1,170 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "lib/translator/read_capacity_10.h"
+#include "gtest/gtest.h"
+
+namespace {
+
+class ReadCapacity10Test : public ::testing::Test {
+ protected:
+  scsi::ReadCapacity10Command read_capacity_10_cmd_;
+  nvme::GenericQueueEntryCmd identify_cmds_[1];
+  absl::Span<nvme::GenericQueueEntryCmd> nvme_cmds_;
+  absl::Span<const uint8_t> scsi_cmd_;
+  nvme::IdentifyNamespace identify_ns_;
+
+  uint8_t buffer_[200];
+
+  // Per-test-suite set-up.
+  // Called before the first test in this test suite.
+  static void SetUpTestSuite() {
+    auto alloc_callback = [](uint16_t count) -> uint64_t {
+      // TODO: fix count == 0
+      // EXPECT_EQ(1, count);
+      return 1337;
+    };
+    void (*dealloc_callback)(uint64_t, uint16_t) = nullptr;
+    translator::SetAllocPageCallbacks(alloc_callback, dealloc_callback);
+  }
+
+  // You can define per-test set-up logic as usual.
+  virtual void SetUp() {
```

Comment:
```suggestion
  void SetUp() override {
```
Methods should only be marked `virtual` if some other class is later going to override them. Conversely, if you're intending to override a method, you should mark it `override`; this both clarifies your intent to a reader, and allows the compiler to warn you if for whatever reason you aren't actually overriding anything.